fableous-fe/src/containers/HubCanvasPage.tsx

```diff
@@ -186,14 +189,17 @@ export default function HubCanvasPage() {
       ControllerRole.Story,
       ControllerRole.Character,
       ControllerRole.Background,
-    ].every((role) => role in joinedControllers);
+    ].some((role) => role in joinedControllers);
```

Comment:
just want to check whether this is commited intentionally or not, Im fine with either using `.every()` or `.some()`

Reply:
fixed